README.md

```diff
@@ -1,13 +1,13 @@
 # Policy publisher
 
-The policy publisher exists to create and manage policy areas and policy programmes
+The policy publisher exists to create and manage policies and sub-policies
 through the Publishing 2.0 pipeline.
 
 ## Nomenclature
 
 - **Policy area**: a broad overview of an area of government activity eg [domestic energy](https://www.gov.uk/government/policies/helping-households-to-cut-their-energy-bills).
```

Comment:
This should be being changed to "Policy" instead of "Policy area", too, I think.